src/index.js

```diff
@@ -28,12 +31,20 @@ async function action() {
       testMode,
       artifactsPath,
       useHostNetwork,
+      createCheck,
       customParameters,
     });
   } finally {
     // Set output
     await Output.setArtifactsPath(artifactsPath);
   }
+
+  if (createCheck) {
+    const fail = await ResultsCheck.publishResults(artifactsPath, checkName, githubToken);
+    if (fail > 0) {
```

Comment:
```suggestion
    const numberOfFailedTests = await ResultsCheck.publishResults(artifactsPath, checkName, githubToken);
    if (numberOfFailedTests >= 1) {
```

Reply:
A few other variables and methods could use more descriptive names as well.

Reply:
You're right that the code could use some cleanup.
I'll do that while I'm writing the unit tests.
In general I prefer `XCount` over `numberOfFailedX`, so how do you feel about the name `failedTestCount`?

Reply:
Sure, I have no real preference as long as a new developer is able to read and understand it.